cmd/version.go

```diff
@@ -34,8 +34,8 @@ func getVersionCmd() *cobra.Command {
 		Use:   "version",
 		Short: "Show application version",
 		Long:  `Show the application version and exit.`,
-		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("k6 v" + consts.FullVersion())
+		Run: func(_ *cobra.Command, _ []string) {
+			fmt.Println("k6 v" + consts.FullVersion()) //nolint:forbidigo // we probably shouldn't do that though
```

Comment:
`--version` is definitely more Unixy, though `<cmd> version` is a Go (Plan 9?) standard AFAIK. I definitely prefer the Unix approach, and that's what most CLI users would expect. A slight issue is that the shorthand `-v` stands for `--verbose`, but that discrepancy is common as well, as a lot of tools just define the long form `--version` for this if `-v` is already taken.

That said, we have a _lot_ of discussion here on GH and on the forum that mentions `k6 version` and it's probably used in CI pipelines as well, so just for backwards compatibility reasons I vote for leaving this as is.

Reply:
>  so just for backwards compatibility reasons I vote for leaving this as is.

Yeah I don't think we can remove `k6 version`. 

Adding `--version` might be a good idea though. @yorugac can you open an issue so we can discussthis outside of this PR :pray:

Reply:
I'm not sure we should do it, for the following motivations:
* It's a breaking change of the CLI API
* We already know that we will migrate to something else, and this seems a dependency on Cobra. It could be different with another library and with could not get the same benefit (so we should at least migrate before).
* As you mentioned, we have `-v` and it could be considered as a shortened version and create confusion

Reply:
Issue is up: https://github.com/grafana/k6/issues/2352
Thank you for responses! Resolving this now.